security.txt

```diff
@@ -0,0 +1,10 @@
+---
+layout: none
+permalink: .well-known/security.txt
```

Comment:
Does this work if you create an actual `.well_known` directory and stick it in there instead?

Reply:
It doesn't pick up `.well-known`, probably because it sees it as a "hidden file"

Reply:
I think it should work if you add to `.include` like this: https://github.com/Homebrew/brew.sh/blob/master/_config.yml#L4

I really don't care particularly though so feel free to CBA my request 😁

Reply:
Seeing how little I know about Jeckyll and that this works as expected I'd rather not bother with that.